pkg/tools/cli/loadtest/config/config.go

```diff
@@ -115,6 +115,7 @@ type Data struct {
 	Concurrency         int                `json:"concurrency" yaml:"concurrency"`
 	BatchSize           int                `json:"batch_size" yaml:"batch_size"`
 	ProgressDuration    string             `json:"progress_duration" yaml:"progress_duration"`
+	ResolveDNS          bool               `json:"resolve_dns" yaml:"resolve_dns"`
```

Comment:
We already have this option in `config.GRPCClient.ConnectionPool.ResolveDNS`

Reply:
Thank you for your comment.
I fixed it.